autopilot/agent.go

```diff
@@ -567,10 +567,15 @@ func (a *Agent) openChans(availableFunds btcutil.Amount, numChans uint32,
 
 	// As channel size we'll use the maximum channel size available.
 	chanSize := a.cfg.Constraints.MaxChanSize()
-	if availableFunds-chanSize < 0 {
+	if availableFunds < chanSize {
 		chanSize = availableFunds
 	}
 
+	if chanSize < a.cfg.Constraints.MinChanSize() {
```

Comment:
We could move this up to avoid iterating through the whole graph in the event that we don't have enough funds.

Reply:
It is already checked before calling `openChans`, this is just an additional sanity check :P